### PR TITLE
make pr page support `revision` field from podutils

### PR DESCRIPTION
**Pull request by krzyzacy**
So that the pr page can support both started and finished.json. (ref https://gubernator.k8s.io/pr/test-infra/10677)

(@ibzib - indeed I need to fix this :joy: it is already broken.. haven't actually deployed a new gubernator yet but will do after this one)

/assign @cjwagner @ibzib

### CODE REVIEW

**Comment by ibzib** [comment]
This is also an issue with the Spyglass PR history page (which is still invisible): 
https://prow.k8s.io/pr-history/kubernetes/test-infra/10677
https://github.com/ibzib/test-infra/blob/3c02dd8802e7a982d6e9b9733f9a279b52283293/prow/cmd/deck/job_history.go#L300

**Comment by krzyzacy** [comment]
(will make a separate fix for spyglass)

**Comment by krzyzacy** [comment]
@stevekuznetsov what reason?

**Comment by stevekuznetsov** [comment]
https://github.com/kubernetes/test-infra/pull/10683#issuecomment-453177993

**Comment by krzyzacy** [comment]
removed started['revison'], this should now match the metadata definition in https://github.com/kubernetes/test-infra/pull/10703, and we can retire the keys in started once we retire bootstrap (or we might retire gubernator before that, who knows)

/hold cancel

**Comment by krzyzacy** [comment]
this should align with #10703 

@ibzib PTAL :-)

File: gubernator/pull_request_test.py [hunk 1]
```diff
@@ -25,6 +25,10 @@ def make(number, version, result, start_time=1000):
     finished = result and {'result': result}
     return (number, started, finished)
 
+def makePodutil(number, version, result, start_time=1000):
```
Comment (ibzib) [inline gubernator/pull_request_test.py]: Nit: `version` -> `revision` for naming consistency

Comment (ibzib) [reply gubernator/pull_request_test.py]: otherwise lgtm

Comment (krzyzacy) [reply gubernator/pull_request_test.py]: done :+1:

**Pull request merged**